resources/hpxml-measures/workflow/tests/compare.py

```diff
@@ -57,7 +56,51 @@ def results(self, base_folder, feature_folder, export_folder, groupby=[]):
         df = df.astype(int)
 
       df = df.fillna('NA')
-      df.to_csv(os.path.join(export_folder, file))
+      df.to_csv(os.path.join(export_folder, file))    
+
+      # Get results charactersistics of groupby columns
+      if file == 'results_characteristics.csv':
+        if 'build_existing_model.geometry_building_type_recs' not in groupby:
```

Comment:
@aspeake Are you able to make this generic so that you could aggregate by any column in results_characteristics.csv (not just geometry_building_type_recs)?

Reply:
@joseph-robertson it is currently set up to always group by RECS building type, but also allows any additional groupby fields to be added alongside building type. I figured it makes sense to default building type, but can make that require an input as well, thoughts?

Reply:
I'd suggest limiting the number of groupby columns to 1. I wouldn't default to any groupbys here since this script is stored on OS-HPXML.